ezgui/src/runner.rs

```diff
@@ -322,18 +322,30 @@ pub fn run<G: 'static + GUI, F: FnOnce(&mut EventCtx) -> G>(settings: Settings,
             EventLoopMode::InputOnly => {
                 running = false;
                 *control_flow = winit::event_loop::ControlFlow::Wait;
-            }
-            EventLoopMode::Animation => {
-                // If we just unpaused, then don't act as if lots of time has passed.
-                if !running {
-                    last_update = Instant::now();
-                    *control_flow = winit::event_loop::ControlFlow::WaitUntil(
-                        Instant::now() + UPDATE_FREQUENCY,
-                    );
-                }
+            },
+            EventLoopMode::Animation(type_str) => {
+                if type_str == "update"{
+                    // If we just unpaused, then don't act as if lots of time has passed.
+                    if !running {
+                        last_update = Instant::now();
+                        *control_flow = winit::event_loop::ControlFlow::WaitUntil(
+                            Instant::now() + UPDATE_FREQUENCY,
+                        );
+                    }
 
-                running = true;
-            }
+                    running = true;
+                }else{
+                    // If we just unpaused, then don't act as if lots of time has passed.
```

Comment:
This branch is the same. github sent me a notification about this PR; are you ready for review or is this in-progress?

Reply:
This is still in-progress